src/_common/video/player/controller.ts

```diff
@@ -53,31 +62,56 @@ export function toggleVideoPlayback(player: VideoPlayerController) {
 	}
 }
 
-export function setVideoVolume(player: VideoPlayerController, level: number) {
+export function setVideoVolume(
```

Comment:
I think we should keep this function very simple as just a way to set the direct video volume. I think we should have a new function that is maybe scrubVideoVolume that works similar to how we scrub the current time. That can do the proper setting of fallback volume and toggling of the video muted state. It would be a function that calls into this function but also does extra work depending on the scrubbing state.